asterius/app/ahc-ar.hs

```diff
@@ -0,0 +1,78 @@
+-- |
+-- Module      :  Main
+-- Copyright   :  (c) 2018 EURL Tweag
+-- License     :  All rights reserved (see LICENCE file in the distribution).
+--
+-- This module contains the implementation of @ahc-ar@, a system-agnostic,
+-- partial implementation of GNU @ar@ for creating archive files from a set of
+-- object files. The shape of the contents of the generated archive files is as
+-- explained [here](https://en.wikipedia.org/wiki/Ar_(Unix)#File_header). For
+-- all fields (except the file size), we use default values, as follows:
+--
+-- > ---------+-----------------------------+----------------+
```

Comment:
Nice diagram here, but the module header documentation is now out-of-date I think, so still better remove it (or change it to mention we use `Ar` in GHC API to generate gnu ar files)

Reply:
revamped the comments: https://github.com/tweag/asterius/pull/663/commits/73643c6e0d50780a846ff10b5bb0d33c7e2b731f